<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Echange externe</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(253, 255, 249);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>941</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>client</string>
     </attribute>
     <widget class="QGroupBox" name="page1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>891</width>
        <height>431</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;MS Reference Sans Serif&quot;;</string>
      </property>
      <property name="title">
       <string>client</string>
      </property>
      <widget class="QLabel" name="nom">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 10pt &quot;Yu Gothic UI Semibold&quot; underlined;
text-decoration: underline;
font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;</string>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_prenom">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 10pt &quot;Yu Gothic UI Semibold&quot; underlined;
text-decoration: underline;
font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;</string>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_cin">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 10pt &quot;Yu Gothic UI Semibold&quot; underlined;
text-decoration: underline;
font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;</string>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_date">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 10pt &quot;Yu Gothic UI Semibold&quot; underlined;
text-decoration: underline;
font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;</string>
       </property>
       <property name="text">
        <string>Date de naissance</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nom">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>140</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(189, 189, 189);

border-width:2px;
border-color:black;
border-style:solid;
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_prenom">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>170</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(189, 189, 189);

border-width:2px;
border-color:black;
border-style:solid;
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_cin">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>200</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(189, 189, 189);

border-width:2px;
border-color:black;
border-style:solid;
</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>230</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(189, 189, 189);

border-width:2px;
border-color:black;
border-style:solid;
</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_model">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>260</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(189, 189, 189);

border-width:2px;
border-color:black;
border-style:solid;
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 10pt &quot;Yu Gothic UI Semibold&quot; underlined;
text-decoration: underline;
font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;</string>
       </property>
       <property name="text">
        <string>Model choisi</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_achat">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>290</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(189, 189, 189);

border-width:2px;
border-color:black;
border-style:solid;
</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>290</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 10pt &quot;Yu Gothic UI Semibold&quot; underlined;
text-decoration: underline;
font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;</string>
       </property>
       <property name="text">
        <string>Achat</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>320</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 10pt &quot;Yu Gothic UI Semibold&quot; underlined;
text-decoration: underline;
font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;</string>
       </property>
       <property name="text">
        <string>Paiement</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_paiement">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>320</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(189, 189, 189);

border-width:2px;
border-color:black;
border-style:solid;
</string>
       </property>
      </widget>
      <widget class="QPushButton" name="terminer">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>360</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(189, 189, 189);

</string>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_text">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>190</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_pic">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>131</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string>l</string>
       </property>
      </widget>
      <widget class="QTableView" name="tabclient">
       <property name="geometry">
        <rect>
         <x>325</x>
         <y>40</y>
         <width>551</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="exportpdf">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>390</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(189, 189, 189);</string>
       </property>
       <property name="text">
        <string>export excel</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>280</y>
         <width>441</width>
         <height>131</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>donner le CIN à chercher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="cin_chercher">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>60</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>30</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(189, 189, 189);</string>
        </property>
        <property name="text">
         <string>chercher</string>
        </property>
       </widget>
       <widget class="QPushButton" name="supprimer_client">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>60</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(189, 189, 189);

</string>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="modifier">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>90</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(189, 189, 189);

</string>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>fournisseur</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>881</width>
        <height>411</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 9pt &quot;MS Reference Sans Serif&quot;;</string>
      </property>
      <property name="title">
       <string>Fournisseur</string>
      </property>
      <widget class="QLabel" name="NomFournisseur">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="NomFournisseur_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(189, 189, 189);

border-width:2px;
border-color:black;
border-style:solid;</string>
       </property>
      </widget>
      <widget class="QLabel" name="piece_demand">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Pièce demandée</string>
       </property>
      </widget>
      <widget class="QLabel" name="nbrePiece">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Nombre</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="nbre">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>160</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(189, 189, 189);

border-width:2px;
border-color:black;
border-style:solid;</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="prenomfour">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(189, 189, 189);

border-width:2px;
border-color:black;
border-style:solid;</string>
       </property>
      </widget>
      <widget class="QLabel" name="datelim">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Date limite</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="datelim_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>190</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(189, 189, 189);

border-width:2px;
border-color:black;
border-style:solid;</string>
       </property>
      </widget>
      <widget class="QLabel" name="prixpayer_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>prix à payer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="prixpayer">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>220</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(189, 189, 189);

border-width:2px;
border-color:black;
border-style:solid;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ajouterFour">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>290</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(189, 189, 189);</string>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="quitter">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>380</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(189, 189, 189);</string>
       </property>
       <property name="text">
        <string>quitter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_prenomfour">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_cinfour">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 9pt &quot;Yu Gothic UI Semibold&quot;;
text-decoration: underline;</string>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="cinfour">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>100</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(189, 189, 189);

border-width:2px;
border-color:black;
border-style:solid;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="piecedemand">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>130</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(189, 189, 189);

border-width:2px;
border-color:black;
border-style:solid;</string>
       </property>
      </widget>
      <widget class="QTableView" name="tabfour">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>40</y>
         <width>531</width>
         <height>201</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(0, 0, 0);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>320</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(189, 189, 189);</string>
       </property>
       <property name="text">
        <string>trier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>350</y>
         <width>101</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(189, 189, 189);</string>
       </property>
       <property name="text">
        <string>envoyer email</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>260</y>
         <width>431</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLineEdit" name="cincherf">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>donner le CIN à chercher</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>20</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;MS Reference Sans Serif&quot;;
background-color: rgb(189, 189, 189);</string>
        </property>
        <property name="text">
         <string>chercher</string>
        </property>
       </widget>
       <widget class="QPushButton" name="modifierFour">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>50</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(189, 189, 189);</string>
        </property>
        <property name="text">
         <string>modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="supprimer_fournisseur">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>80</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(189, 189, 189);</string>
        </property>
        <property name="text">
         <string>supprimer</string>
        </property>
       </widget>
      </widget>
      <zorder>groupBox_2</zorder>
      <zorder>NomFournisseur</zorder>
      <zorder>NomFournisseur_2</zorder>
      <zorder>piece_demand</zorder>
      <zorder>nbrePiece</zorder>
      <zorder>nbre</zorder>
      <zorder>prenomfour</zorder>
      <zorder>datelim</zorder>
      <zorder>datelim_2</zorder>
      <zorder>prixpayer_2</zorder>
      <zorder>prixpayer</zorder>
      <zorder>ajouterFour</zorder>
      <zorder>quitter</zorder>
      <zorder>label_prenomfour</zorder>
      <zorder>label_cinfour</zorder>
      <zorder>cinfour</zorder>
      <zorder>piecedemand</zorder>
      <zorder>tabfour</zorder>
      <zorder>pushButton_2</zorder>
      <zorder>pushButton_3</zorder>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
